modules/internal/DefaultRectangular.chpl

```diff
@@ -306,11 +306,11 @@ module DefaultRectangular {
         chpl_debug_writeln("### numTasksPerLoc = ", numTasks, "\n",
                            "### ignoreRunning = ", ignoreRunning, "\n",
                            "### minIndicesPerTask = ", minIndicesPerTask, "\n",
-                           "### numChunks = ", numChunks, " (parDim = ", parDim, ")\n",
+                           "### numChunks = ", numChunks, " (parDim = ", parDim+1, ")\n",
```

Comment:
I'd leave this the same

Reply:
Good point.  This (and the two following instances) are addressed in commit: https://github.com/chapel-lang/chapel/pull/14522/commits/3dbb1e630c66a7b3432f1349841034f57a4baa69